Editors/ImportExportEditor/Test.ImportExport/Importing/Importers/GltfToRmvImporter/GltfIToRmvImporterTests.cs

```diff
@@ -0,0 +1,121 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Editors.ImportExport.Importing.Importers.GltfToRmv;
+using Editors.ImportExport.Importing.Importers.GltfToRmv.Helper;
+using Moq;
+using Shared.Core.Events;
+using Shared.Core.PackFiles;
+using Shared.Core.PackFiles.Models;
+using Shared.Core.Services;
+using Shared.GameFormats.RigidModel;
+using Shared.TestUtility;
+using Shared.Ui.BaseDialogs.StandardDialog;
+using SharpGLTF.Schema2;
+using Test.ImportExport.Exporting.Exporters.RmvToGlft;
+
+namespace Test.ImportExport.Importing.Importers.GltfImporterTest
+{
+    class TestData
+    {
+        public static readonly string InputGtlfFile = PathHelper.FileFromDataFolder(@"Karl_Gltf_Resaved_From_Blender\karl_franz_imported_and_saved_to_blendfile_and_exported_to_gltf_from_blender.gltf");
+        public static readonly string InputPack = PathHelper.FileFromDataFolder("Humanoid01Skeleton_Pack");
```

Comment:
Could you not use the default test pack with karl? it contains that skeleton? 
Its preferable to limit the amount of test data